pkg/server/controllers.go

```diff
@@ -621,6 +621,52 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 		return err
 	}
 
+	controllerName = "permission-claim-label-controller"
```

Comment:
Honestly not sure about decoupling these like this, but I'm not familiar of another way to insert a post start hook func & name.